deployment.template.yaml

```
@@ -32,11 +32,18 @@ spec:
                 secretKeyRef:
                   name: $CI_PROJECT_NAME-config
                   key: mnemonic
+          readinessProbe:
+            httpGet:
+              path: /ready
+              port: 5555
+            failureThreshold: 1
+            periodSeconds: 30
+            initialDelaySeconds: 60
```

I think we can keep it tight here.

```suggestion
            periodSeconds: 10
            initialDelaySeconds: 60
```